interfaces/default/js/dash.js

```diff
@@ -172,6 +172,33 @@ function loadRecentMovies() {
   })
 }
 
+function loadRecentMoviesList() {
+  if (!$('#latestmovieslist_table_body').length) return
+  $.getJSON(WEBDIR + 'kodi/GetRecentMovies', function(result) {
+    if (result.movies[0] == null) {
+      $('#latestmovieslist_table_body').append(
+        $('<tr>').append($('<td>').html('No latest movies found').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2')),
+        $('<tr>').append($('<td>').html('&nbsp;').attr('colspan', '2'))
+      )
+      return
+    }
+    $.each(result.movies, function(i, item) {
+
+      var fanart = WEBDIR + 'kodi/GetThumb?h=240&w=430&thumb=' + encodeURIComponent(item.fanart)

```

Comment:
This seem to be unused. Please remove it. I'll and merge your or later
Today but it looks good. Thanks for your contribution.